packages/design-system/src/components/AskAssistantChat/AskAssistantChat.vue

```diff
@@ -70,12 +70,12 @@ function isEndOfSessionEvent(event?: ChatUI.AssistantMessage) {
 }
 
 function onQuickReply(opt: ChatUI.QuickReply) {
-	emit('message', opt.text, opt.type);
+	emit('message', opt.text, opt.type, opt.isFeedback);
 }
 
 function onSendMessage() {
 	if (sendDisabled.value) return;
-	emit('message', textInputValue.value, undefined);
+	emit('message', textInputValue.value, undefined, undefined);
```

Comment:
is not this the same? 
```suggestion
	emit('message', textInputValue.value);
```

Reply:
it is, we need to update event definition in `emits` so ts doesn't complain. Will do that